tests/core_tests/CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ set(core_tests_sources
   integer_overflow.cpp
   multisig.cpp
   ring_signature_1.cpp
-  service_nodes.cpp
+  loki_tests.cpp
```

Comment:
This is not a big deal, but I kind of think the `service_nodes.cpp` name fits better here (or maybe it could be `loki_service_nodes.cpp`) as I think we are likely to want to add other tests here in the future and don't want to suggest that we want to cram all loki-specific tests into one file.

Reply:
Happy to down the road as long as we don't end up files with 1 test in them like block_reward, chain_split, chain_switch etc. Too sprawly otherwise.